fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/scan_config/cli/cmd/FoDScanConfigSetupSastCommand.java

```diff
@@ -50,14 +50,13 @@
 @DisableTest(TestType.CMD_DEFAULT_TABLE_OPTIONS_PRESENT)
 public class FoDScanConfigSetupSastCommand extends AbstractFoDJsonNodeOutputCommand implements IRecordTransformer, IActionCommandResultSupplier {
```

Comment:
I'd like to review this command in more detail, but don't have time. Maybe best to just hide/mark as preview either this command only, or the full `scan-config` command tree?

Reply:
I don't think we should hide this one but I will mark it as PREVIEW